ignite/engine/engine.py

```diff
@@ -118,18 +117,18 @@ def compute_mean_std(engine, batch):
 
     """
 
-    _state_dict_all_req_keys = ("epoch_length", "max_epochs")
-    _state_dict_one_of_opt_keys = ("iteration", "epoch")
+    _state_dict_all_req_keys = ("epoch_length",)
+    _state_dict_one_of_opt_keys = (("iteration", "epoch",), ("max_epochs", "max_iters",))
```

Comment:
We can relax the constraint that the both `max_epochs` and `max_iters` could not have values.